rustls/src/conn.rs

```diff
@@ -257,6 +271,43 @@ mod connection {
         }
     }
 
+    impl BufRead for Reader<'_> {
+        /// Obtain a chunk of plaintext data received from the peer over this TLS connection.
```

Comment:
Where are all these docs coming from? Ideally we'd reference the other place rather than making copies at this level of detail (which easily falls out of sync).

Reply:
Fair enough, it was copied from `Read::read` / `Read::read_buf` methods (which were already copy pasted)

Reply:
Should remove this from the original commit.